### ISSUE TITLE
CDN link is not working

### ISSUE THREAD

**Issue by cheungvong** [ReduxFramework/elusive-icons#39]
The link to CDN is giving an access denied. <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/elusive-icons/2.0.0/css/elusive-icons.min.css">

**Comment by GwynethLlewelyn** [comment]
Indeed. It seems that as BootstrapCDN moved to Stackpath (or were acquired by them), something went wrong when the URL changed...
